src/components/CommentsContainer.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import { MdSend } from "react-icons/md";
+import { useDispatch } from "react-redux";
+import { Link } from "react-router-dom";
+import { addComment } from "../services/comments/commentsService";
+import CommentSection from "./CommentSection";
```

Comment:
Suggestion: You can import from index.js to avoid multiple import statements...or try to use absolute path method

Reply:
It's in the same folder